compiler/main/checks.cpp

```diff
@@ -470,6 +470,15 @@ static void check_afterResolveIntents()
       // Only look at Var or Arg symbols
       if (isLcnSymbol(sym)) {
         QualifiedType qual = sym->qualType();
+        // MPF TODO: This should not be necessary
+        // it is a workaround for problems with --verify
+        // with tuple type constructors accepting domains.
+        // It would be better to treat run-time types a
+        // normal record.
```

Comment:
Typo in this comment.  Should be "better to treat runtime types *as* normal *records*"?